build-support/pants_venv

```diff
@@ -35,8 +35,14 @@ function remove_venv() {
 }
 
 function create_venv() {
+  if command -v "${PY}" >/dev/null; then
+    PY="$(command -v "${PY}")"
+    log "Using python at ${PY}"
+  else
+    die "No ${PY} interpreter found on the path. Python will not work!"
```

Comment:
The `venv_dir` function assumes `$PY` is defined and runs before this, so in the end this is just code-noise that misleads the reader into thinking we'll give a nice message. We may or may not, it's hard to trace.

A follow-up might be able to clean this up by requiring the only public function - `activate_pants_venv` - be passed the python interpreter as a parameter or else default it and from there pass it to the rest of the private functions here that need it.